### PR TITLE
PERF-5402: Convert shard-lite FLE variant to 3-Node

### PR DESCRIPTION
**Pull request by jawwadasghar**
[PERF-5402](https://jira.mongodb.org/browse/PERF-5402): Convert shard-lite FLE variant to 3-Node

Converting FLE shard lite to shard.
Manually editing docs to get passed failed self test in [related DSI PR](https://github.com/10gen/dsi/pull/2046)

[7.0 Patch](https://spruce.mongodb.com/version/670553c9f049bf00071eb7e5/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)
[8.0 Patch](https://spruce.mongodb.com/version/6705364e77c87f0007a32030/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)
[Master Patch](https://spruce.mongodb.com/version/670534dccb2efd0007ed57b7/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC)

### CODE REVIEW

**Review by PyGeek03 — APPROVED** [review]
My team is deferring this PR to product perf.

File: evergreen/system_perf/5.0/genny_tasks.yml [hunk 1]
```diff
@@ -43,7 +43,6 @@ buildvariants:
   - name: multi_updates_multi_updates
 - name: perf-3-node-replSet.arm.aws.2023-11
   tasks:
-  - name: device_monitoring
```
Comment (mbor-dev) [inline evergreen/system_perf/5.0/genny_tasks.yml:46]: Why are we removing device_monitoring? I don't see this as part of the task.

Comment (jawwadasghar) [reply evergreen/system_perf/5.0/genny_tasks.yml:46]: That was removed form the PrivateWorkloads repo as part of [PERF-5941](https://jira.mongodb.org/browse/PERF-5941). 
Task lists in genny are now getting auto updated based on that removal.

**Review by mbor-dev — APPROVED** [review]
LGTM

**Comment by alicedoherty** [comment]
We need to update the [AutoRun conditions](https://github.com/mongodb/genny/blob/master/src/workloads/encrypted/YCSBLikeQueryableEncrypt1Cf16.yml#L43) from `shard-lite-fle` to `shard-fle`, otherwise when someone (e.g., me as I just found out) tries to regenerate the Genny task definitions these changes will be overwritten. I will fix this as part of a PR for [BF-35328](https://jira.mongodb.org/browse/BF-35328).

**Pull request merged**